e2e/test/scenarios/question/summarization.cy.spec.js

```diff
@@ -93,16 +93,15 @@ describe("scenarios > question > summarize sidebar", () => {
       toBinning: "10 bins",
     });
 
-    H.getDimensionByName({ name: "Total" }).should(
-      "have.attr",
-      "aria-selected",
-      "true",
-    );
-    H.getDimensionByName({ name: "Quantity" }).should(
-      "have.attr",
-      "aria-selected",
-      "true",
-    );
+    H.getDimensionByName({ name: "Total" })
+      .scrollIntoView()
```

Comment:
These 2 dimensions are selected but this one is before the next one, so we need to scroll this one into view, so we can see both dimension in the same view.